dhcpv6/dhcpv6_test.go

```diff
@@ -150,5 +150,20 @@ func TestNewReplyFromRebind(t *testing.T) {
 	require.Equal(t, rep.Type(), REPLY)
 }
 
+func TestNewSolicitWithCID(t *testing.T) {
```

Comment:
you should also check for the presence of `OptClientID`, and use a non-zero-value DUID, so the tests fails if it's not set to the right value